.github/CONTRIBUTING.md

```diff
@@ -195,6 +195,16 @@ python -m pip install ".[dev, examples]"
 python -m pip install pre-commit
 ```
 
+Additionally, for testing backward compatibility with older versions of Pytorch Lightning, you also need to download all saved version-checkpoints from the public AWS storage. Run the following script to get all saved version-checkpoints:
```

Comment:
```suggestion
Additionally, for testing backward compatibility with older versions of PyTorch Lightning, you also need to download all saved version-checkpoints from the public AWS storage. Run the following script to get all saved version-checkpoints:
```

Reply:
I would like to also suggest to formulate it as optional. I believe to get started with contributing this is *absolutely not necessary*.

Reply:
@awaelchli - in that case, this should be reflected with `make test` too: https://github.com/PyTorchLightning/pytorch-lightning/blob/master/.github/CONTRIBUTING.md#developments-scripts 

or `make test` should not run legacy checkpoint tests